src/zope/site/__init__.py

```diff
@@ -20,10 +20,28 @@
 from zope.site.site import threadSiteSubscriber
 from zope.site.site import clearThreadSiteSubscriber
 
+__all__ = [
+    'SiteManagerContainer',
+    'SiteManagementFolder',
+    'SiteManagerAdapter',
+    'LocalSiteManager',
+    'threadSiteSubscriber',
+    'clearThreadSiteSubscriber',
+    'changeSiteConfigurationAfterMove',
+]
+
 
 # BBB. Remove in Version 5.0
 from zope.component import getNextUtility
 from zope.component import queryNextUtility
 from zope.deprecation import deprecated
-getNextUtility = deprecated(getNextUtility, '``zope.site.getNextUtility`` is deprecated and will be removed in zope.site 5.0. Use it from ``zope.component`` instead.')  # noqa
-queryNextUtility = deprecated(queryNextUtility, '``zope.site.queryNextUtility`` is deprecated and will be removed in zope.site 5.0. Use it from ``zope.component`` instead.')  # noqa
+getNextUtility = deprecated(
+    getNextUtility,
+    '``zope.site.getNextUtility`` is deprecated '
+    'and will be removed in zope.site 5.0. '
+    'Use it from ``zope.component`` instead.')  # noqa
+queryNextUtility = deprecated(
+    queryNextUtility,
+    '``zope.site.queryNextUtility`` is deprecated '
+    'and will be removed in zope.site 5.0. '
+    'Use it from ``zope.component`` instead.')  # noqa
```

Comment:
Is this `# noqa` still needed?

Reply:
🤷 I have no idea. It didn't seem to do anything as it was, i.e., flake8 was still complaining about the line being too long. I can try taking it out and seeing what happens.